src/Components/Pages/LoginPage.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import LoginForm from './LoginForm/LoginForm'
+
+export default function LoginPage() {
+  return (
+    <div className='LoginPageContainer'>
```

Comment:
```suggestion
    <div>
```

pas besoin de rajouter de nom de class ici surtout si tu n'associe pas de CSS dedans